buidl/test/test_hd.py

```diff
@@ -408,6 +413,30 @@ def test_from_mnemonic(self):
             private_key = HDPrivateKey.from_mnemonic(mnemonic, b"TREZOR")
             self.assertEqual(private_key.xprv(), xprv)
 
+    def test_from_dice(self):
+        tests = [  # dice_rolls, expected_mnemonic, expected_xprv
+            [
+                "",
+                "together mail awful cradle scrub apart hip leader silk slice unusual embark kit can muscle nature nation gown century cram resource citizen throw produce",
+                "xprv9s21ZrQH143K2adDUydUhh9twPi4jxoG4dPHKsVNcid2DJRbpfUuemEETVYhr91RGSQHvyED6REErWA19bLG4iRrbWPCFsNdMWswBgABd2N",
+            ],
+            [
+                "123456",
+                "mirror reject rookie talk pudding throw happy era myth already payment own sentence push head sting video explain letter bomb casual hotel rather garment",
+                "xprv9s21ZrQH143K3cCzWRegMLRXHDc3NgQCNWZY5XqPG4Qo7ZjF5QMM1J8NN64d3mikoRr9PzLVG6Sm7iJQF3p91thHsWcyxHZ4ENXd9vz8C4r",
+            ],
+            [
+                "123456123456123456123456123456123456123456123456123456123456123456123456123456123456123456123456123456",
+                "more matter caught bind tip twin indicate visa rifle angle defense lizard stock cave cradle injury always mule photo horse range opinion affair garlic",
+                "xprv9s21ZrQH143K4J7isGyg4whNPAMGdfNntamxzPk2qFDS4nNJtNwrZ6vaiv6Jk5JsNyXKfZT5X3QnEMaTg3uvcKszhdgbLxgMh6ETE3nrzPn",
```

Comment:
Is this what coldcard outputs?

Reply:
Can you add [this test vector](https://www.keepitsimplebitcoin.com/coldcards-dice-roll-math/) for `523365252662366`? It was the only other one I could find online.

Reply:
Sure thing.  Added in 6de06301bbfc795e8474bca57d1d3b019c5c8da0.